clients/drcachesim/tests/offline-tlb_simulator_v2p.templatex

```diff
@@ -0,0 +1,31 @@
+#ifdef AARCH64
+TLB simulation results:
+Core #0 \(1 thread\(s\)\)
+  L1I stats:
+    Hits:                      *[0-9,\.]*
+    Misses:                    *[0-9,\.]*
+    Compulsory misses:         *[0-9,\.]*
+    Invalidations:             *0
+    Miss rate:                        [0-3][,\.]..%
+  L1D stats:
+    Hits:                      *[0-9,\.]*
```

Comment:
Hmm, does this prove the TLB simulator used the physical instead of the virtual addresses?  If I disabled the reading of the file would this template still match?

Do we need a unit test to really check which addresses are being used instead of end-to-end?

Reply:
Working on a unit test for TLB simulator, I realized we didn't have any!

Reply:
(Just a note: https://github.com/DynamoRIO/dynamorio/pull/6961#issuecomment-2343652883 issue has been fixed)

Reply:
> Working on a unit test for TLB simulator, I realized we didn't have any!

Unfortunately yes, more technical debt...will be good to have it added.

Reply:
Done.